posts/2022-10-06-memory-footprint-throughput-update.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "How does Open Liberty’s performance compare to other cloud-native Java runtimes"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/gcharters
+author_github: https://github.com/gcharters
+seo-title: How does Open Liberty’s performance compare to other cloud-native Java runtimes - OpenLiberty.io
+seo-description: While cold startup time is critically important for cloud functions (and Open Liberty's time-to-first-response is approximately 1 second, so it's no slouch), memory footprint and throughput are far more important for microservices architectures in which each running instance is likely to serve thousands of requests before it's replaced.
+blog_description: "While cold startup time is critically important for cloud functions (and Open Liberty's time-to-first-response is approximately 1 second, so it's no slouch), memory footprint and throughput are far more important for microservices architectures in which each running instance is likely to serve thousands of requests before it's replaced."
+open-graph-image: https://openliberty.io/img/blog/perfblog3a.png
+additional_authors:
+- name: Tim Pickett
+  github: https://github.com/tpickett1
+  image: https://avatars0.githubusercontent.com/tpickett1
+---
+= How does Open Liberty’s performance compare to other cloud-native Java runtimes
+Graham Charters <https://github.com/gcharters>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+For many years, the most important metric for application servers was throughput: how many transactions or requests can we squeeze out of this application?  The adoption of finer-grained architecture styles, such as microservices and functions, has led to a broadening in the performance characteristics we should care about, such as _memory footprint_ and _throughput_.
+
+Where you once deployed a small number of runtime instances for your server and application, now you may be deploying tens or hundreds of microservices or functions.  Each of those instances comes with its own server, even if it's embedded in a runnable jar or pulled in as a container image there's a server in there somewhere. While cold startup time is critically important for cloud functions (and link:{url-prefix}/blog/2019/10/30/faster-startup-open-liberty.html[Open Liberty's time-to-first-response is approximately 1 second], so it's no slouch), memory footprint and throughput are far more important for microservices architectures in which each running instance is likely to serve thousands of requests before it's replaced.
```

Comment:
```suggestion
Where you once deployed a small number of runtime instances for your server and application, now you might deploy tens or hundreds of microservices or functions.  Each of those instances comes with its own server. Even if it's embedded in a runnable jar or pulled in as a container image, there's a server in there somewhere. While cold start time is critically important for cloud functions (and link:{url-prefix}/blog/2019/10/30/faster-startup-open-liberty.html[Open Liberty's time-to-first-response is approximately 1 second], so it's no slouch), memory footprint and throughput are far more important for microservices architectures in which each running instance is likely to serve thousands of requests before it's replaced.
```

Reply:
```suggestion
Where you once deployed a small number of runtime instances for your server and application, now you might deploy tens or hundreds of microservices or functions.  Each of those instances comes with its own server.  Even if it's embedded in a runnable jar or pulled in as a container image there's a server in there somewhere. While cold startup time is critically important for cloud functions (and link:{url-prefix}/blog/2019/10/30/faster-startup-open-liberty.html[Open Liberty's time-to-first-response is approximately 1 second], so it's no slouch), memory footprint and throughput are far more important for microservices architectures in which each running instance is likely to serve thousands of requests before it's replaced.
```

Reply:
Other changes are fine but I think we should keep the reference to 'startup time' to be called startup rather than start since linked blog post has 'startup' in multiple places.  The term 'startup' is regularly used in our WAS performance charts, presentations, etc.